crates/rpc-types-eth/src/filter.rs

```diff
@@ -894,6 +894,35 @@ impl From<Vec<Transaction>> for FilterChanges {
     }
 }
 
+impl<T> FilterChanges<T> {
+    /// Get the hashes if present.
+    pub fn hashes(&self) -> Option<&[B256]> {
```

Comment:
repo standard api is 

is_hashes -> bool
as_hashes -> Option<>

Reply:
fixed